### PR TITLE
refresh ordebook on interval. 

### PR DESCRIPTION
**Pull request by bthaile**
set refreshing order book every 3 seconds, remove full loading of market. call get market info when user goes to market page.

### CODE REVIEW

File: packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ import { OutcomeOrderBook } from 'modules/types';
 import { createBigNumber } from 'utils/create-big-number';
 import { formatShares } from 'utils/format-number';
 
+const ORDDER_BOOK_REFRESH = 3000;
```
Comment (robert-e-davidson3) [inline packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx]:
```suggestion
const ORDDER_BOOK_REFRESH_MS = 3000;
```

Comment (robert-e-davidson3) [reply packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx]: nit

Comment (bconfortin) [reply packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx]: Also: typo on "orDDer"

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx [hunk 2]
```diff
@@ -159,7 +159,7 @@ export default class MarketView extends Component<
 
     const {
       isConnected,
-      loadFullMarket,
+      loadMarketsInfo,
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:162]: Is that the specific info for a *single* market? If so, I suggest `loadMarketInfo`, singular.

**Review by bconfortin — CHANGES_REQUESTED** [review]
There is a typo on `ORDDER_BOOK_REFRESH` and I agree with Robert on adding `_MS` to the end, so we know it's in milliseconds.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 💚
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/654hvnq**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:162]: that is the method name used over the app. going to leave it.

File: packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx]: nice one, will fix

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market/components/market-view/market-view.tsx:162]: also it take an array

**Comment by bthaile** [comment]
PR updated with suggested review changes.

**Review by bconfortin — APPROVED** [review]
Lgtm

**Pull request merged**